hack/lib/util.sh

```diff
@@ -437,6 +437,23 @@ kube::util::git_upstream_remote_name() {
     head -n 1 | awk '{print $1}'
 }
 
+# Ensures the current directory is a git tree for doing things like restoring or
+# validating godeps
+kube::util::ensure-git-tree() {
```

Comment:
`ensure` implies that this is idem-potent, doesn't it?

Also `kube::util::ensure-fake-git-tree` or something would make it less misleading.